telegram/ext/filters.py

```diff
@@ -2560,3 +2561,70 @@ def filter(self, message: Message) -> bool:
 
 VOICE = _Voice("filters.VOICE")
 """Messages that contain :attr:`telegram.Message.voice`."""
+
+
+class Mention(MessageFilter):
```

Comment:
Need to move this class since the objects in this file are ordered alphabetically